lib/jnpr/junos/device.py

```diff
@@ -702,7 +702,7 @@ def cli(self, command, format='text', warning=True):
             if rsp is True:
                 return ''
             if rsp.tag in ['output', 'rpc-reply']:
-                return rsp.text
+                return etree.tostring(rsp, method="text")
```

Comment:
Can you change this to 

return etree.tounicode(rsp, method="text", with_tail=False)

etree.tostring wont be compatible with python 3.x